CMakeLists.txt

```diff
@@ -400,6 +406,8 @@ if(ENABLE_MASON)
     set(ENV{CCACHE_CPP2} "true")
   endif()
 
+  include_directories(SYSTEM ${CMAKE_CURRENT_SOURCE_DIR}/third_party/libosmium/include)
```

Comment:
Can we add a note here that `find_package` is not compatible with `mason_use` hence we need to manually define the include path here?